pom.xml

```diff
@@ -347,6 +347,11 @@
       <artifactId>arangodb-java-driver</artifactId>
       <version>6.9.0</version>
     </dependency>
+    <dependency>
```

Comment:
Perhaps in this PR, you could only add the questdb dependency. Updating the other dependencies might have some unintended side effects (e.g., CI test failures).

Reply:
It seems all the checks pass, so it might be fine to keep the updates.

Reply:
CI test was the only one failed. Anyways I reverted back the version bump.